Update docs wording to match Deploy & Workflow renaming (#254)
statelyai/docs: docs/stately-sky-getting-started.mdx

## Step 3: Deploy your machine to Sky

> Once you have generated the API key, you can deploy your machine to Sky as a running workflow.

Reviewer: ```suggestion
Once you have generated the API key, you can deploy your machine to Sky as a workflow.
```

1. Use the **Deploy to Sky 🌤️** button to start the deployment process.
2. When the workflow is deployed, it will be listed under **Existing deploys**.
3. Use **Copy URL** to copy to the URL to your clipboard.

> You’ll need the running workflow’s URL to communicate with it from the starter project.

Reviewer: ```suggestion
You’ll need the workflow’s URL to communicate with it from the starter project.
```

![.env file](../static/assets/sky-getting-started/code-env-file.png)

## Step 5: Add the actor in the starter project

> After adding the API key, you’ll need to add the actor.

Reviewer: ```suggestion
After adding the API key, you’ll need to create an actor.
```

import { actorFromStately } from '@statelyai/sky';

const actor = actorFromStately({
  url: 'paste your Sky url here',
  sessionId: 'your session id here',
});
```

:::tip
By default, Sky is multiplayer.

> The session ID is used to shard the multiplayer session.

Reviewer: ```suggestion
The session ID is used to shard the multiplayer session. Each actor has a unique session ID.
```

Author: Should this be "Each workflow has a unique session ID."?

> - Use a shared session ID to allow multiple users to reference the same state.

Reviewer: ```suggestion
- Use a shared session ID to allow multiple users to reference the same actor.
```

Author: should this be "reference the same workflow."?

- If you want users isolated, use a unique session ID for each user.

:::

## Step 6: Fetching the config from Sky

> Now that we’ve added the actor, we need to fetch the config from Sky. Doing so will download and generate the machine configuration file in our repo, giving us type safety when interacting with the running actor!

Reviewer: ```suggestion
Now that we’ve created the actor, we need to fetch the config from Sky. Doing so will download and generate the machine configuration file in our repo, giving us type safety when interacting with the running actor!
```
